UI/window-basic-main.cpp

```diff
@@ -5261,6 +5261,20 @@ void OBSBasic::changeEvent(QEvent *event)
 			if (previewEnabled)
 				EnablePreviewDisplay(true);
 		}
+	} else if (event->type() == QEvent::ActivationChange) {
+		bool enablePreviewToggle =
+			config_get_bool(App()->GlobalConfig(), "BasicWindow",
+					"WindowFocusTogglePreview");
+
+		if (!isActiveWindow()) {
+			previewEnabledWhileFocused = previewEnabled;
+
+			if (enablePreviewToggle) {
+				DisablePreview();
+			}
+		} else if (enablePreviewToggle && previewEnabledWhileFocused) {
+			EnablePreview();
```

Comment:
You probably want to use `EnablePreviewDisplay` like the code above as these functions do not work in Studio mode.

Reply:
Ah yeah makes sense thanks. Updated